src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtil.java

```diff
@@ -468,6 +468,17 @@ public static URI getResourceFromClassPath(String filename) throws CheckstyleExc
         return uri;
     }
 
+    /**
```

Comment:
It is not possible to mock `Class.getResource` so we need a helper method.
This method can me mocked.